packages/strapi-plugin-content-manager/admin/src/utils/removeKeyInObject.js

```diff
@@ -20,7 +20,11 @@ const removeKeyInObject = (obj, keyToRemove) => {
       return { ...acc, [current]: value.map(obj => removeKeyInObject(obj, keyToRemove)) };
     }
 
-    if (typeof acc[current] === 'object') {
+    if (typeof value === 'object') {
+      if (value._isAMomentObject === true) {
```

Comment:
Can you add a test for the following statement?

Thanks a lot for your contribution

Reply:
For sure, thank you a lot, bro!

Reply:
Hey, I have added test for this specific case.